rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.adoc

```diff
@@ -8,25 +8,36 @@ toc::[]
 
 [NOTE]
 ====
-If you are looking for a quickstart guide for ROSA, see xref:../rosa_getting_started/rosa-quickstart-guide-ui.adoc#rosa-quickstart-guide-ui[{product-title} quickstart guide].
+If you are looking for a quickstart guide for ROSA classic, see xref:../rosa_getting_started/rosa-quickstart-guide-ui.adoc#rosa-quickstart-guide-ui[{product-title} quickstart guide].
 ====
 
-{hcp-title-first} offers a more efficient and reliable architecture for creating ROSA clusters. With {hcp-title}, each cluster has a dedicated control plane that is isolated in a Red Hat account.
+{hcp-title-first} offers a more efficient and reliable architecture for creating ROSA clusters. With {hcp-title}, each cluster has a dedicated control plane that is isolated in a ROSA service account.
+
+:FeatureName: {hcp-title-first} 
+include::snippets/technology-preview.adoc[]
+
+Create {hcp-title} cluster quickly by using the default options and automatic AWS Identity and Access Management (IAM) resource creation. You can deploy your cluster by using the ROSA CLI (`rosa`).
 
 [IMPORTANT]
 ====
-Since it is not possible to "upgrade" to a {hcp} architecture, you must create a new cluster to benefit from the for {hcp-title} functionality.
+Since it is not possible to "upgrade" or "convert" existing ROSA clusters to a {hcp} architecture, this architecture requires a new cluster to benefit from the {hcp-title} functionality.
```

Comment:
Also, this sentence seems a bit awkward - maybe something like:

"Since it is not possible to upgrade or convert existing ROSA clusters to a {hcp} architecture, you must create a new cluster to use {hcp-title} functionality."